### ISSUE TITLE
Pasting HTML into inline editor clones the element

### ISSUE THREAD

**Issue by infosaurus** [tinymce/tinymce#3497]
I have the following inline editable element :

`<h2 class="editable-simple mce-content-body" id="title" style="position: relative;" contenteditable="true" spellcheck="false" data-postback="Title">title</h2>`

initialized with

```
        tinymce.init({
            selector: '.editable-simple',
            inline: true,
            toolbar: 'undo redo',
            menubar: false,
            plugins: "paste",
            paste_as_text: true
        });
```

In Internet Explorer, when I copy a heading from somewhere else (either a h1, h2, etc. from a web page or a heading from Word) and paste it in that editor, the `<h2>` element is duplicated. In other words, a second `<h2>` is added to the DOM and appears besides the normal one. It is editable but blank (doesn't contain pasted text) and has the same ID and attributes as the real one. 

Sometimes, for instance when I repeatedly paste the heading in the editor,  the same thing occurs but the inserted h2 is non-editable and contains pasted text.

`paste_as_text: true` is an attempt to go around that problem by preventing formatted text to be pasted, but doesn't solve anything.

Thanks for your help.

TinyMCE 4.5.1
Internet Explorer 11.0

**Comment by fyrkant** [comment]
Hi! Can you reproduce the issue in a [fiddle](http://fiddle.tinymce.com/) to make it easier for us to understand the problem?

**Comment by infosaurus** [comment]
Hi fyrkant,

I reproduced the issue in fiddle.tinymce.com by just inserting the code from my first message and copy pasting in the inline editor a h1 from any web page, say a Wikipedia article for instance.

Repeating it here, just for reference : 

```
<script type="text/javascript">
        tinymce.init({
            selector: '.editable-simple',
            inline: true,
            toolbar: 'undo redo',
            menubar: false,
            plugins: "paste",
            paste_as_text: true
        });
</script>

<h2 class="editable-simple mce-content-body" id="articleTitle" style="position: relative;" contenteditable="true" spellcheck="false" data-postback="Title">sqsds</h2>
```

![image](https://cloud.githubusercontent.com/assets/1136870/23133133/598afc72-f790-11e6-8dda-dec2b8038b18.png)

**Comment by infosaurus** [comment]
Anyone? Extra stuff needed besides the fiddle?

Thanks

**Comment by infosaurus** [comment]
Anyone have any idea why this is still in "needs more info"?

Please let me know if extra details are needed , thanks.

**Comment by metricjs** [comment]
Hi,
I'm unable to reproduce this issue in TinyMCE 4.9.10 and 5.2.2 using the details provided, however looking back at the changelog it appears this issue was fixed in TinyMCE 4.7.12. As such I'm going to close this, however if you're still experiencing issues then let us know and we'll reopen this issue.
Thanks!
